editor/editor_inspector.cpp

```diff
@@ -833,30 +833,41 @@ void EditorProperty::_update_pin_flags() {
 	}
 }
 
-Control *EditorProperty::make_custom_tooltip(const String &p_text) const {
-	tooltip_text = p_text;
+static Control *make_help_bit(const String &p_text, bool p_property) {
+	PackedStringArray slices = p_text.split("::", false);
+	if (slices.is_empty()) {
+		// Not a valid EditorHelpBit text, use default tooltip.
+		return nullptr;
+	}
```

Comment:
That's an unverified assumption, but the previous behavior was to return an `EditorHelpBit` with an empty `RichTextLabel`, which is probably never useful. Now the `nullptr` return is handled by `Viewport::_gui_show_tooltip` to create a default tooltip using the same `p_text` as here.